.github/workflows/start-release.yml

```diff
@@ -430,6 +430,9 @@ jobs:
       - run: echo "${{needs.get-version.outputs.version}}"
       - name: Notify Success
         run: |
+          curl "${{ secrets.ZAPIER_WEBHOOK_URL }}" \
```

Comment:
What does this do? A comment would be nice, it's not obvious from the code what is the effect of the command. Does it post something on Zapier? What is the URL where we can see the pushed information?

Reply:
Addressed - thanks!